frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -222,6 +224,9 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
         state.bucketClassType === BucketClassType.STANDARD
           ? !!state.tier1Policy
           : !!state.namespacePolicyType,
+      canJumpTo:
+        stepHigherBound >=
+        Object.values(CreateStepsBC).indexOf(CreateStepsBC[CreateStepsBC.PLACEMENT]) + 1,
```

Comment:
You can calculate `Object.values(CreateStepsBC)` outside and reuse the same in all the steps.